packages/environment-check/src/checks/environment.ts

```diff
@@ -44,6 +48,69 @@ export async function getEnvironment(): Promise<{ environment: Environment; mess
     };
 }
 
+/**
+ * Returns the VSCode environment.
+ *
+ * @returns environment, including tools, extensions ...
+ */
+export async function getToolsExtensions(): Promise<{
+    toolsExtensions: ToolsExtensions;
+    messages: ResultMessage[];
+}> {
+    const logger = getLogger();
+
+    const extensions = await getInstalledExtensions();
```

Comment:
this step could fail, but should not cause the whole check to fail. Perhaps wrap it in `try/catch` or add try catch inside `getInstalledExtensions`